### PR TITLE
Wait for the previous build to complete before starting

### LINKED ISSUE #3065: Figure out how to prevent concurrent tests from running on the test servers

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Fixes #3065

### CODE REVIEW

File: tests/wait_until_previous_builds_complete.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DRONE_BUILD_NUM=${DRONE_BUILD_NUM:=0}
+prevBuildStatus=`drone build info vmware/vic $(( $DRONE_BUILD_NUM-8 ))`
+outArray=($prevBuildStatus)
+
+while [[ ${outArray[2]} == *"running"* ]]; do
```
Comment (mlh78750) [inline tests/wait_until_previous_builds_complete.sh]: I'm weak on shell. This does re-execute the drone command to update the state, right?

**Comment by mlh78750** [comment]
couple of small comments.. 

LGTM

**Comment by caglar10ur** [comment]
LGTM

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**